libs/designer/src/lib/ui/panel/templatePanel/templatePanel.tsx

```diff
@@ -24,11 +28,17 @@ export const TemplatePanel = ({ onCreateClick }: { onCreateClick: (onSuccessfulC
     dispatch(clearTemplateDetails());
   }, [dispatch]);
   const createWorkflowPanelTabs = useCreateWorkflowPanelTabs({
-    onCreateClick: (onSuccessfulCreation: () => void) => onCreateClick(onSuccessfulCreation),
+    onCreateClick: () => onCreateClick(),
   });
   const currentPanelTabs: TemplatePanelTab[] = useMemo(
-    () => (currentPanelView === 'createWorkflow' ? createWorkflowPanelTabs : getQuickViewTabs(intl, dispatch)),
-    [currentPanelView, createWorkflowPanelTabs, intl, dispatch]
+    () =>
+      currentPanelView === 'createWorkflow'
+        ? createWorkflowPanelTabs
+        : getQuickViewTabs(intl, dispatch, {
+            templateId: templateName ?? '',
```

Comment:
I think instead of empty string '', lets pass Unknown so we can query the logs to see what we are missing

Reply:
?? 'Unknown'